### PR TITLE
style singular posts

### PR DESCRIPTION
**Pull request by claireclark1**
Purpose
---------------
feature

Solution
---------------
style the post card according to wireframes: https://app.zeplin.io/project/60a3f1172759472a903908f7/screen/61003f320349c51086d0fc63

- no formatting of the grid, just the individual posts
- delete login quickstart
- delete reply input
- do not fix timestamp

Change summary
---------------
- changes mentioned above
- mock attachment data
- onHover, display timestamp and actions block

Steps to Verify
----------------
1. refer to wire-frames
2. check that styling matches.

Additional details / screenshot
----------------
Screen Shot 2021-08-02 at 2 19 39 PM

### CODE REVIEW

File: src/components/Post.tsx [hunk 1]
```diff
@@ -1,26 +1,41 @@
...
+  //   (Array.isArray(noteContent.attachment)
+  //     ? noteContent.attachment
+  //     : [noteContent.attachment]);
+
+  const attachments: any = [
+    {
+      type: "Image",
+      url: "https://www.industrialempathy.com/img/remote/ZiClJf-1920w.jpg",
+    },
+  ];
```
Comment (claireclark1) [inline src/components/Post.tsx]: mock earlier on?

File: src/components/Post.tsx [hunk 2]
```diff
@@ -29,14 +44,20 @@ const Post = ({ feedItem }: PostProps): JSX.Element => {
...
-      <PostReply />
+        {showActionsBar && <ActionsBar timestamp={feedItem.timestamp} />}
+      </div>
+      <div className="Post__caption">
+        <div>{noteContent.content}</div>
+        <div className="Post__captionTags">
+          {feedItem.tags && feedItem.tags[0]}
+        </div>
+      </div>
+      {/*<PostReply />*/}
```
Comment (claireclark1) [inline src/components/Post.tsx]: delete component all together

File: src/components/scss/Profile.scss [hunk 3]
```diff
@@ -1,4 +1,4 @@
-.ProfileBlock__block {
+.Profile__block {
```
Comment (shannonwells) [inline src/components/scss/Profile.scss:1]: Thanks for getting rid of "profile block block" 😸

**Review by shannonwells — APPROVED** [review]
Works on my machine!

**Pull request merged**